webapp/components/admin_console/log_settings.jsx

```diff
@@ -112,7 +112,7 @@ export default class LogSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.log.fileDescription'
-                            defaultMessage='Typically set to true in production.  When true, log files are written to the log file specified in file location field below.'
+                            defaultMessage='Typically set to true in production.  When true, log files are written to the mattermost.log file specified in the location field below.'
```

Comment:
This might be better:

*Typically set to true in production.  When true, logged events are written to the mattermost.log file in the directory specified in the File Log Directory field. The logs are rotated at 10,000 lines and archived to a file in the same directory, and given a name with a datestamp and serial number. For example, mattermost.2017-03-31.001.*